### DISCUSSION TITLE
0018 XLS-18d Standard For Bootstrapping XRPLD Networks

### DISCUSSION THREAD

**Discussion by RichardAH** [XRPLF/XRPL-Standards#42] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by mvadari** [comment]
@RichardAH should this spec be moved into PR and kept in the repo or should it be closed? Note that this does not mean that the spec has been accepted/implemented/finalized.

I will move it to PR by copy-pasting the existing draft if there is no response in the next week.
